tools/clearcdncache

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+
+host=$1
+dns_provider_url=$2
```

Comment:
no need, this will only work with Cloudflare

Reply:
I should remove this file?

Reply:
I should remove this file?

Reply:
not the whole file, but this parameter, dns_provider_url should be a constant in this file

Reply:
You shouldn't need to pass it from the caller of this script

Reply:
You mean should be 
`dns_provider_url="https://api.cloudflare.com/client/v4/zones/${CF_ZONE}"`